go_to_office/lib/screens/admin/office.dart

```diff
@@ -99,8 +95,8 @@ class _OfficePageState extends State<OfficePage> {
   }
 
   Future<bool> updateOffice() {
-    if (repository != null) {
-      return repository.updateOffice(this.office);
+    if (FirebaseRepository != null) {
```

Comment:
Are you sure this null check is good?
Don't we need to check FirebaseRepository.firebaseRepository?